### DISCUSSION TITLE
v2.0.0 Beta 3

### DISCUSSION THREAD

**Discussion by Kovah** [Kovah/LinkAce#848] [Announcements]
<p align="center">
<img src="https://github.com/user-attachments/assets/7fb07843-1ab9-42d9-867a-c8b536f3c455" alt="LinkAce v2 Beta 3"/>
</p>

### This is the third BETA release of LinkAce 2 including SSO support

Changes from v2.0 Beta 2 to Beta 3:

- Support for Single Sign On via OAuth or OIDC with various providers built-in. See [the SSO setup documentation](https://www.linkace.org/docs/v2/configuration/sso-oauth-oidc/) for enabling SSO in this Beta.
- New translation: Dutch (thanks to [lindely](https://crowdin.com/profile/lindely)!)
- Update of dependencies including security patches.

**Full Changelog**: https://github.com/Kovah/LinkAce/compare/v2.0.0-beta2...v2.0.0-beta3

<hr /><em>This discussion was created from the release <a href='https://github.com/Kovah/LinkAce/releases/tag/v2.0.0-beta3'>v2.0.0 Beta 3</a>.</em>

---

Please have a look at the [Beta 1 release notes](https://github.com/Kovah/LinkAce/discussions/832) for more details about the Beta phase of LinkAce 2.

To upgrade from Beta 1 to Beta 2, follow the standard upgrade guide for LinkAce.

**Comment by MAngel666** [comment]
Is it now possible to sync links with a browser?

**Reply by Kovah** [reply]
Nop and it's nothing I would like to build into LinkAce itself. You may create your own browser extension which syncs to LinkAce via its API.
